src/core/net/ip6.hpp

```diff
@@ -349,6 +349,11 @@ class Ip6 : public InstanceLocator, private NonCopyable
      */
     void ResetBorderRoutingCounters(void) { ClearAllBytes(mBorderRoutingCounters); }
 #endif
+    /**
+     * Removes matching MPL entry.
+     *
+     */
+    bool RemoveMplEntry(Message &aMessage) { return mMpl.RemoveMatchedMessage(aMessage); }
```

Comment:
We don't use the `bool` return value. Should we make `void`?

Reply:
sure, modified to void